src/world/area_kmr/kmr_20/8EBE50.c

```diff
@@ -2,11 +2,20 @@
 
 #include "world/common/atomic/Pipe.inc.c"
 
-INCLUDE_ASM(s32, "world/area_kmr/kmr_20/8EBE50", func_80240310_8EC130);
+ func_80240310_8EC130(void) {
+    set_map_transition_effect(4);
+    return 2;
+}
 
-INCLUDE_ASM(s32, "world/area_kmr/kmr_20/8EBE50", func_80240330_8EC150);
+ func_80240330_8EC150(void) {
+    func_8005566C(0, 0x3E8, 0x3F);
```

Comment:
Can you declare this function in functions.h? i.e. add `void func_8005566C(s32, s32, s32);` to it